services/viva/microservice/test/lambdas/syncWorkflow.test.ts

```diff
@@ -0,0 +1,145 @@
+import { GetWorkflowResult, LambdaRequest, syncWorkflow } from '../../src/lambdas/syncWorkflow';
+import { EncryptionType } from '../../../types/caseItem';
+import { CaseItem } from '../../src/types/caseItem';
+
+const PK = 'USER#199001011234';
+const SK = 'CASE#11111111-2222-3333-4444-555555555555';
+const WORKFLOW_ID = 'flowId123ABC';
+
+const caseKeys = {
+  PK,
+  SK,
+};
+
+const input: LambdaRequest = {
+  detail: {
+    user: {
+      personalNumber: '199801011234',
+    },
+  },
+};
+
+function createWorkflow(): GetWorkflowResult {
```

Comment:
I think there is a lot of information returned by the function that isn't needed. I think it can be refactored to:

```js
function createWorkflow(): GetWorkflowResult {
  return {
    attributes: {
      workflowid: WORKFLOW_ID,
      application: {
        completiondate: null,
        receiveddate: null,
        requestingcompletion: null,
        // or whichever properties that is used 
      },
    },
  } as GetWorkflowResult;
}
```